machine-learning/voting-classifier.ipynb

```diff
@@ -0,0 +1,292 @@
+    "You can view the dashboard by clicking the link after running the cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<table style=\"border: 2px solid white;\">\n",
+       "<tr>\n",
+       "<td style=\"vertical-align: top; border: 0px solid white\">\n",
+       "<h3>Client</h3>\n",
+       "<ul>\n",
+       "  <li><b>Scheduler: </b>tcp://127.0.0.1:39415\n",
+       "  <li><b>Dashboard: </b><a href='http://127.0.0.1:8787/status' target='_blank'>http://127.0.0.1:8787/status</a>\n",
+       "</ul>\n",
+       "</td>\n",
+       "<td style=\"vertical-align: top; border: 0px solid white\">\n",
+       "<h3>Cluster</h3>\n",
+       "<ul>\n",
+       "  <li><b>Workers: </b>4</li>\n",
+       "  <li><b>Cores: </b>4</li>\n",
+       "  <li><b>Memory: </b>4.02 GB</li>\n",
+       "</ul>\n",
+       "</td>\n",
+       "</tr>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "<Client: scheduler='tcp://127.0.0.1:39415' processes=4 cores=4>"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import dask_ml.joblib\n",
+    "from sklearn.externals import joblib\n",
+    "from distributed import Client\n",
+    "\n",
+    "client = Client()\n",
+    "client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, train the same model but using dask."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "VotingClassifier(estimators=[('sgd', SGDClassifier(alpha=0.0001, average=False, class_weight=None, epsilon=0.1,\n",
+      "       eta0=0.0, fit_intercept=True, l1_ratio=0.15,\n",
+      "       learning_rate='optimal', loss='hinge', max_iter=1000, n_iter=None,\n",
+      "       n_jobs=1, penalty='l2', power_t=0.5, random_state=None,\n",
+      "       shuffle=T...,\n",
+      "  max_iter=-1, probability=False, random_state=None, shrinking=True,\n",
+      "  tol=0.001, verbose=False))],\n",
+      "         flatten_transform=None, n_jobs=-1, voting='hard', weights=None)\n",
+      "CPU times: user 172 ms, sys: 30.2 ms, total: 202 ms\n",
+      "Wall time: 1.19 s\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%time \n",
+    "with joblib.parallel_backend(\"dask\", scatter=[X, y]):\n",
+    "    clf.fit(X, y)\n",
+    "\n",
+    "print(clf)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "I see no advantage of using dask because sklearn is already using all my computers processors, but dask enables users to take advantage of cluster computing and train models across multiple different machines."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Record software used in this notebook."
```

Comment:
I have mixed thoughts about this. It's nice from a reproducibility point of view, but I think it's a bit distracting from the message. I think curious users will be able to find this repo, with the exact binder image and all that.